lib/sms.rb

```diff
@@ -0,0 +1,7 @@
+class SMS
```

Comment:
I think your sense that this should be it's own class was a good one.  It would cut down on the responsibilities currently held in the Takeaway class, and have the added effect of making properly testing the functionality of sending an sms easier.